CODIGO/Declares.bas

```diff
@@ -734,6 +749,11 @@ Type Inventory
     PuedeUsar As Byte
 End Type
 
+Public Type t_UserInvetory
```

Comment:
Ideally I would move this into a class.

Reply:
same as above, I'm already doing a lot of work here i dont want to refactor the whole project